packages/apps/ability-relay-link/test/e2e/eoa.spec.ts

```diff
@@ -0,0 +1,415 @@
+import {
+  setupVincentDevelopmentEnvironment,
+  getEnv,
+  type PkpInfo,
+  type VincentDevEnvironment,
+} from '@lit-protocol/vincent-e2e-test-utils';
+import {
+  disconnectVincentAbilityClients,
+  getVincentAbilityClient,
+} from '@lit-protocol/vincent-app-sdk/abilityClient';
+import * as util from 'node:util';
+import { JsonRpcProvider } from 'ethers-v6';
+import {
+  serializeTransaction,
+  hexToBigInt,
+  hexToNumber,
+  type Hex,
+  type TransactionSerializable,
+} from 'viem';
+
+import {
+  bundledVincentAbility as relayLinkAbility,
+  getRelayLinkQuote,
+  toVincentTransaction,
+  type Transaction,
+} from '../../src';
+
+jest.setTimeout(300000);
+
+describe('Relay.link Ability E2E Tests', () => {
+  const ENV = getEnv();
```

Comment:
```suggestion
  const ENV = getEnv(
    {
      BASE_RPC_URL: z.string(),
    }
  );
```

`BASE_RPC_URL` isn't apart of what `getEnv` parses